annotation-file-utilities/src/annotator/find/ParamCriterion.java

```diff
@@ -10,8 +10,8 @@
 
 public class ParamCriterion implements Criterion {
 
-  private final String methodName;
-  private final Integer paramPos;
+  final String methodName;
```

Comment:
Please write brief documentation for these fields.  (For example, is the integer index zero-based or one-based?